apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/cross-chain/data-table.tsx

```diff
@@ -95,6 +95,8 @@ export function DataTable({
     "Successfully deployed contract",
     "Failed to deploy contract",
   );
+  const [isAddingRow, setIsAddingRow] = useState(false);
+  const [addRowError, setAddRowError] = useState<string | null>(null);
```

Comment:
You can remove these status by just making adding `handleAddRow` in a `useMutation` hook - and then use `.isPending` and `.error` from mutation result